src/ophys_etl/workflows/ophys_experiment.py

```diff
@@ -85,3 +91,114 @@ def from_id(
             session=session,
             specimen=specimen
         )
+
+    def get_ophys_experiment_motion_border(self,
+            session: Session) -> Dict:
+        """
+        Get motion border for an ophys experiment
+
+        Parameters
+        ----------
+        ophys_experiment_id
+            The ophys experiment id
+        session
+            The database session
+
+        Returns
+        -------
+        Dict[int]
+            A dictionary containing motion border data
+        """
+
+        workflow_step_run_id = get_latest_run(session,
+                                              WorkflowStepEnum.MOTION_CORRECTION,
+                                              WorkflowNameEnum.OPHYS_PROCESSING,
+                                              )
+        query = (
+            select(
+                MotionCorrectionRun,
+            )
+            .where(MotionCorrectionRun.workflow_step_run_id == workflow_step_run_id) 
+        )
+
+        result = session.execute(query).all()
+        motion_border = result[0][0]
+        return {
+            "x0": motion_border.max_correction_left,
+            "x1": motion_border.max_correction_right,
+            "y0": motion_border.max_correction_up,
+            "y1": motion_border.max_correction_down
+        }
+
+    def _generate_binary_mask(self, ophys_roi: OphysROI, ophys_roi_mask_values: List[OphysROIMaskValue]) -> List[List[bool]]:
```

Comment:
since this is not specific to ophys experiment, but to an roi, I would prefer if this were placed in a new file `utils/roi_utils.py`